src/js/pannellum.js

```diff
@@ -378,8 +380,14 @@ function init() {
         
         if (config.dynamic !== true) {
             // Still image
+            if (config.panorama instanceof Image) {
```

Comment:
Also check for instance of `ImageData` or `ImageBitmap` here?

Reply:
Done.